Library/Formula/perl.rb

```diff
@@ -35,7 +35,15 @@ def install
 
     system "./Configure", *args
     system "make"
-    system "make", "test" if build.with?("tests") || build.bottle?
+
+    # OS X El Capitan's SIP feature prevents DYLD_LIBRARY_PATH from being passed to child
+    # processes, which causes the make test step to fail.
+    # https://rt.perl.org/Ticket/Display.html?id=126706
+    # https://github.com/Homebrew/homebrew/issues/41716
+    if MacOS.version < :el_capitan
+      system "make", "test" if build.with?("tests") || build.bottle?
```

Comment:
Will merge as-is, but if it bothers you I won't sulk if you want to change it :smile:.